doc/man/scons.xml

```diff
@@ -2156,8 +2177,57 @@ repositories are searched in the order specified.</para>
 
 <refsect1 id='configuration_file_reference'>
 <title>CONFIGURATION FILE REFERENCE</title>
-<!--  .SS Python Basics -->
-<!--  XXX Adding this in the future would be a help. -->
+
+<refsect2 id='configuration_files'>
+<title>Configuration Files</title>
+<para>
+The build configuration consists of one or more configuration files
+(SConscript files).
+The main file is named <filename>SConstruct</filename> by
+default, though if necessary, &scons; also checks for alternate spellings
+&Sconstruct;, &sconstruct;, &SConstruct.py;, &Sconstruct.py;
+and &sconstruct.py;
+in that order.
+An alternate file name (including a path to another
```

Comment:
Instead of "An alternate file name" how about "Other file names"..

Reply:
Ok.  A subtle point of `-f` is it changes the algorithm from "look in current directory" to "look in the place I said" if there's a path component, always looking for a better wording to include that without getting too wordy.  Maybe "A different sconstruct file can be specified using the `-f` option"?